assemblies/platform-plugin/src/main/resources/resources/web/common-ui-require-js-cfg.js

```diff
@@ -270,28 +272,28 @@
     "pentaho/visual/samples/calc",
 
     // ccc vizs
-    "pentaho/visual/ccc/abstract",
-    "pentaho/visual/ccc/cartesianAbstract",
-    "pentaho/visual/ccc/categoricalContinuousAbstract",
-    "pentaho/visual/ccc/barAbstract",
-    "pentaho/visual/ccc/barNormalizedAbstract",
-    "pentaho/visual/ccc/barHorizontal",
-    "pentaho/visual/ccc/bar",
-    "pentaho/visual/ccc/barStacked",
-    "pentaho/visual/ccc/barStackedHorizontal",
-    "pentaho/visual/ccc/barNormalized",
-    "pentaho/visual/ccc/barNormalizedHorizontal",
-    "pentaho/visual/ccc/barLine",
-    "pentaho/visual/ccc/line",
-    "pentaho/visual/ccc/metricDotAbstract",
-    "pentaho/visual/ccc/areaStacked",
-    "pentaho/visual/ccc/pie",
-    "pentaho/visual/ccc/heatGrid",
-    "pentaho/visual/ccc/sunburst",
-    "pentaho/visual/ccc/donut",
-    "pentaho/visual/ccc/scatter",
-    "pentaho/visual/ccc/bubble"
-  ].forEach(registerVizPackage);
+    "pentaho/visual/models/abstract",
```

Comment:
This comment should also be changed. Following the change in resources/config.js:
`Pentaho stock visualizations' models`